contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -889,6 +902,7 @@ select
 from pg_proc p
 inner join sys.schemas sch on sch.schema_id = p.pronamespace
 left join pg_trigger tr on tr.tgfoid = p.oid
+left join sys.babelfish_function_ext f on p.proname = f.funcname and sch.schema_id::regnamespace::name = f.nspname
```

Comment:
Yeah there will be multiple rows for the same function if there are overloaded functions but there is an assumption that most of the existing customer applications would be using unique function/procedure names. Otherwise I don't see any field in pg_proc which can be used to store create_date.

Reply:
As per your offline suggestion, I have implemented a helper function `babelfish_get_pltsql_function_signature`. It returns a function's signature which can be used along with function name to get primary key join.